prometheus/config/init.sls

```diff
@@ -1,7 +1,14 @@
 # -*- coding: utf-8 -*-
 # vim: ft=sls
 
+{#- Get the `tplroot` from `tpldir` #}
+{%- set tplroot = tpldir.split('/')[0] %}
+{%- from tplroot ~ "/map.jinja" import prometheus with context %}
+
 include:
   - .users
   - .args
   - .file
+    {%- if 'node_exporter' in prometheus.wanted %}
+  - .node_exporter.textfile_collectors
```

Comment:
Similar to previous comment. The pillar data should map to `prometheus:config:node_exporter` (or even `prometheus:config:exporters:node_exporter` but maybe this is too much)

Reply:
my mistake. this refers to `prometheus.config.node_exporter.textfile_collectors` state.